.travis.yml

```diff
@@ -18,5 +18,9 @@ before_install:
   - grunt build && cd ..
 
 script:
-  - mvn test
-  - cd $UI_DIR && grunt test
+  - mvn test -P postgres
+  - cd $UI_DIR && grunt test && cd ..
```

Comment:
The UI/grunt tests don't execute any java code. They can run once as the last item.